lib/ansible/modules/network/avi/avi_pool.py

```diff
@@ -0,0 +1,374 @@
+        default: True
+    fail_action:
+        description:
+            - Enable an action - close connection, http redirect, local http response, or backup pool - when a pool failure happens.
+            - By default, a connection will be closed, in case the pool experiences a failure.
+    fewest_tasks_feedback_delay:
+        description:
+            - Periodicity of feedback for fewest tasks server selection algorithm.
+        default: 10
+    graceful_disable_timeout:
+        description:
+            - Used to gracefully disable a server.
+            - Virtual service waits for the specified time before terminating the existing connections  to the servers that are disabled.
+        default: 1
+    health_monitor_refs:
+        description:
+            - Verify server health by applying one or more health monitors.
+            - Active monitors generate synthetic traffic from each service engine and mark a server up or down based on the response.
+            - The passive monitor listens only to client to server communication.
+            - It raises or lowers the ratio of traffic destined to a server based on successful responses.
+            - It is a reference to an object of type healthmonitor.
+    host_check_enabled:
+        description:
+            - Enable common name check for server certificate.
+            - If enabled and no explicit domain name is specified, avi will use the incoming host header to do the match.
+        default: False
+    inline_health_monitor:
+        description:
+            - The passive monitor will monitor client to server connections and requests and adjust traffic load to servers based on successful responses.
+            - This may alter the expected behavior of the lb method, such as round robin.
+        default: True
+    ipaddrgroup_ref:
+        description:
+            - Use list of servers from ip address group.
+            - It is a reference to an object of type ipaddrgroup.
+    lb_algorithm:
+        description:
+            - The load balancing algorithm will pick a server within the pool's list of available servers.
+        default: 1
+    lb_algorithm_consistent_hash_hdr:
+        description:
+            - Http header name to be used for the hash key.
+    lb_algorithm_hash:
+        description:
+            - Criteria used as a key for determining the hash between the client and  server.
+        default: 1
+    max_concurrent_connections_per_server:
+        description:
+            - The maximum number of concurrent connections allowed to each server within the pool.
+        default: 0
+    max_conn_rate_per_server:
+        description:
+            - Rate limit connections to each server.
+    name:
+        description:
+            - The name of the pool.
+        required: true
+    networks:
+        description:
+            - (internal-use) networks designated as containing servers for this pool.
```

Comment:
This information is still returned in the API. It is just not editable. In next version of Avi we will clean it up then I would be updating it to deprecated. However, till our next release happens we need to keep it in the object.